File: js/ejer_56.js
// 56. Simular carrito de compras (Agregar productos, mostrar total)
const cart = [
  { product: 'Manzana', price: 2.5, qty: 3 },
  { product: 'Pan', price: 1.2, qty: 2 },
  { product: 'Leche', price: 3.0, qty: 1 }
];
const total = cart.reduce((sum, item) => sum + (item.price * item.qty), 0);
console.log('Carrito:');
cart.forEach(item => console.log(`  ${item.product}: $${item.price} x${item.qty} = $${(item.price * item.qty).toFixed(2)}`));
console.log('Total:', '$' + total.toFixed(2));

File: python/ejer_56.py
# 56. Simular carrito de compras (Agregar productos, mostrar total)
cart = [
    {'product': 'Manzana', 'price': 2.5, 'qty': 3},
    {'product': 'Pan', 'price': 1.2, 'qty': 2},
    {'product': 'Leche', 'price': 3.0, 'qty': 1}
]
total = sum(item['price'] * item['qty'] for item in cart)
print('Carrito:')
for item in cart:
    subtotal = item['price'] * item['qty']
    print(f"  {item['product']}: ${item['price']} x{item['qty']} = ${subtotal:.2f}")
print('Total: $' + str(round(total, 2)))
